### PR TITLE
Fix trends page issues

### PR DESCRIPTION
**Pull request by danangmassandy**
This PR is for below tickets (the preview are in each ticket):

- #1765
- #1751 
- #1740
- #1673
- Fix layout in national trend
- Fix population trends charts that grows outside chart container

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 26 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 50 sec

File: django_project/frontend/src/utils/Helpers.ts [hunk 1]
```diff
@@ -60,3 +62,34 @@ export const isMapDisplayed = () => {
...
   return str.split(' ').map((word) => capitalize(word)).join(' ').trim()
 }
+
+
+/**
+ * Check if user has a Data Consumer role
+ * @param userInfo user info from API
+ * @returns True if user has group/role of Data Consumer
+ */
+export function isDataConsumer(userInfo: UserInfo) {
```
Comment (dimasciput) [inline django_project/frontend/src/utils/Helpers.ts:72]: @danangmassandy let's use permission instead. See here https://github.com/kartoza/sawps/pull/1788#pullrequestreview-1761621040

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 40 sec

File: django_project/frontend/src/utils/Helpers.ts [hunk 2]
```diff
@@ -60,3 +62,34 @@ export const isMapDisplayed = () => {
...
+/**
+ * Format datetime to text
+ * @param dateTime 
+ * @param defaultText text value if dateTime is null
+ * @returns formatted datetime in DD/MM/YYYY hh:mm:ss
+ */
+export function displayDateTime(dateTime: Date, defaultText?: string) {
+  if (dateTime == null && defaultText) return defaultText;
+  if (dateTime == null && !defaultText) return ' ';
+  console.log('dateTime ', dateTime)
```
[the remark this answers is not in the document]
Comment (danangmassandy) [reply django_project/frontend/src/utils/Helpers.ts]: I've removed the console log

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 9 min 32 sec

**Pull request merged**